### PR TITLE
Blackbox: --run-performance-tests

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 

Adds a `--no-performance-tests` flag to BB tests, which allows to skip performance tests.

Add any further explanations here. 

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running BB tests with the flag
* [ ] If applicable, add screenshots or log transcripts of the feature working
* [ ] 
image

### CODE REVIEW

**Review by mssalvatore — CHANGES_REQUESTED** [review]
The changes look good to me. 2 small items

1. This should probably get an entry in the CHANGELOG.md
2. Would it be better to change the option to `--run-performance-tests` and have it default to `False`? Most of the time, I don't want to run these tests, so it seems to me that not running the tests should be the default behavior. I'm not sure if you have a different experience.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Use pipenv for python dependency management. #1091
 - Moved unit tests to a dedicated `tests/` directory to improve pytest
   collection time. #1102
+- Added `--run-performance-tests` flag to BB tests. If this flag is not specified,
```
Comment (mssalvatore) [inline CHANGELOG.md:25]: Either 

1. drop the word "Added" and move this to the `Added` section above

or

2. Reword as a change (e.g. "Performance tests are skipped if BB tests aren't run with the `--run-performance-tests` flag")

**Review by mssalvatore — CHANGES_REQUESTED** [review]
Just a small tweak to the changelog. Otherwise it looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (VakarisZ) [reply CHANGELOG.md:25]: I'd like to keep "Added" for new features. Skipping performance tests is nothing new functionally, just a convenience thing.

**Pull request merged**